src/main/java/com/ss/training/utopia/counter/entity/User.java

```diff
@@ -0,0 +1,111 @@
+package com.ss.training.utopia.counter.entity;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * @author Justin O'Brien
+ */
+@Entity
+@Table(name = "tbl_user")
+public class User implements Serializable {
+
+	private static final long serialVersionUID = -1093118640730467189L;
+
+	@Id
+	@Column
+	private int userId;
+
+	@Column
+	private String name, username, password, role;
```

Comment:
I explicitly name the columns for each variable, but this looks like it isn't necessary. I think I developed that practice from the not wanting the same name in the POJO that was in the library database.